generate.sh

```diff
@@ -6,9 +6,12 @@ if [ ! -e fhir-parser ]; then
 	git submodule update --init
 fi
 
+pip3 install -r generator-requirements.txt
```

Comment:
I have a local setup using a virtual environment, but that's probably not true for everybody. Since the generation script is only really needed by the repo maintainers I think it's okay if it's not a one-stop command.

Can you remove the changes to `generate.sh`? Then I'll see that this gets merged in, thanks!